### ISSUE TITLE
[Bug]: panic: write xxxx: no space left on device in logservice and cn & dn crashed continuously 

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#18812]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch Name

main

### Commit ID

8910a7732b83133999db16e93ccc228429776391

### Other Environment Information

```Markdown
- Hardware parameters:
  3*CN: 16C 64G
  1*DN: 16C 64G
  3*LOG: 4C 16G
  3*PROXY: 3C 7G
- OS type:
- Others:
```

### Actual Behavior

job link:
https://github.com/matrixorigin/mo-nightly-regression/actions/runs/10863533524/job/30155729008

During benchmark regression, log services crash by 'no space left on device' and then caused all of cns and dns were crashed continuously.
[root@VM-0-124-tencentos ~]# kubectl -n mo-main-nightly-8910a7732-20240914 get pod
NAME                                 READY   STATUS             RESTARTS         AGE
nightly-regression-dis-dn-0          1/1     Running            42 (7m9s ago)    15h
nightly-regression-dis-log-0         0/1     CrashLoopBackOff   60 (2m13s ago)   15h
nightly-regression-dis-log-1         0/1     CrashLoopBackOff   60 (107s ago)    15h
nightly-regression-dis-log-3         1/1     Running            0                15h
nightly-regression-dis-proxy-dzgfb   1/1     Running            0                15h
nightly-regression-dis-proxy-t79dm   1/1     Running            0                15h
nightly-regression-dis-tp-cn-4n6bf   1/1     Running            0                15h
nightly-regression-dis-tp-cn-5jnkp   0/1     CrashLoopBackOff   42 (4m19s ago)   15h

{"level":"PANIC","time":"2024/09/15 07:36:00.070170 +0000","name":"dragonboat","caller":"v4@v4.0.0-20240312080931-1b40809d7cea/nodehost.go:2230","msg":"write /var/lib/matrixone/data/logservice-data/00000000-0000-0000-0000-000000000001/nightly-regression-dis-log-1/06166173447481204388/tandb/node-0-131073/000005.idxtmp: no space left on device","stacktrace":"github.com/lni/dragonboat/v4.panicNow\n\t/go/pkg/mod/github.com/matrixorigin/dragonboat/v4@v4.0.0-20240312080931-1b40809d7cea/nodehost.go:2230\ngithub.com/lni/dragonboat/v4.(*NodeHost).startShard\n\t/go/pkg/mod/github.com/matrixorigin/dragonboat/v4@v4.0.0-20240312080931-1b40809d7cea/nodehost.go:1649\ngithub.com/lni/dragonboat/v4.(*NodeHost).StartReplica\n\t/go/pkg/mod/github.com/matrixorigin/dragonboat/v4@v4.0.0-20240312080931-1b40809d7cea/nodehost.go:508\ngithub.com/matrixorigin/matrixone/pkg/logservice.(*store).startHAKeeperReplica\n\t/go/src/github.com/matrixorigin/matrixone/pkg/logservice/store.go:216\ngithub.com/matrixorigin/matrixone/pkg/logservice.(*store).startReplicas\n\t/go/src/github.com/matrixorigin/matrixone/pkg/logservice/store.go:201\ngithub.com/matrixorigin/matrixone/pkg/logservice.NewService\n\t/go/src/github.com/matrixorigin/matrixone/pkg/logservice/service.go:128\nmain.startLogService\n\t/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/main.go:335\nmain.startService\n\t/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/main.go:229\nmain.main\n\t/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/main.go:117\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:272"}
{"level":"INFO","time":"2024/09/15 07:36:00.136461 +0000","caller":"motrace/syncer.go:89","msg":"Wait signal done."}
panic: write /var/lib/matrixone/data/logservice-data/00000000-0000-0000-0000-000000000001/nightly-regression-dis-log-1/06166173447481204388/tandb/node-0-131073/000005.idxtmp: no space left on device

goroutine 1 gp=0xc0000081c0 m=8 mp=0xc0004b0008 [running]:
panic({0x4261bc0?, 0xc008db4050?})
        /usr/local/go/src/runtime/panic.go:804 +0x168 fp=0xc004cf69b0 sp=0xc004cf6900 pc=0x47b268
go.uber.org/zap/zapcore.CheckWriteAction.OnWrite(0x0?, 0xc005c7e100?, {0x0?, 0x0?, 0xc004f280e0?})
        /go/pkg/mod/go.uber.org/zap@v1.24.0/zapcore/entry.go:198 +0x54 fp=0xc004cf69d0 sp=0xc004cf69b0 pc=0x6bddf4
go.uber.org/zap/zapcore.(*CheckWriteAction).OnWrite(0x0?, 0x0?, {0x0?, 0x0?, 0x4c76897?})
        <autogenerated>:1 +0x2d fp=0xc004cf6a08 sp=0xc004cf69d0 pc=0x6ca84d
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc003d220d0, {0x0, 0x0, 0x0})
        /go/pkg/mod/go.uber.org/zap@v1.24.0/zapcore/entry.go:264 +0x24e fp=0xc004cf6b98 sp=0xc004cf6a08 pc=0x6be16e
go.uber.org/zap.(*SugaredLogger).log(0xc0004931e8, 0x4, {0x4c666d9?, 0x0?}, {0xc008f2e410?, 0xc0020de920?, 0xc004cf6c10?}, {0x0, 0x0, 0x0})
        /go/pkg/mod/go.uber.org/zap@v1.24.0/sugar.go:295 +0xec fp=0xc004cf6bd8 sp=0xc004cf6b98 pc=0x89cf0c
go.uber.org/zap.(*SugaredLogger).Panicf(...)
        /go/pkg/mod/go.uber.org/zap@v1.24.0/sugar.go:189
github.com/matrixorigin/matrixone/pkg/logutil.DragonboatAdaptLogger.Panicf(...)
        /go/src/github.com/matrixorigin/matrixone/pkg/logutil/dragonboat.go:65
github.com/matrixorigin/matrixone/pkg/logutil.(*DragonboatAdaptLogger).Panicf(0xc0020de900?, {0x4c666d9?, 0x416e85?}, {0xc008f2e410?, 0x42d52c0?, 0x20001?})

logserivce crash log:
[log.txt](https://github.com/user-attachments/files/17004966/log.txt)

mo log:
https://grafana.ci.matrixorigin.cn/explore?panes=%7B%22CXX%22:%7B%22datasource%22:%22loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bnamespace%3D%5C%22mo-main-nightly-8910a7732-20240914%5C%22%7D%20%7C%3D%20%60panic%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki%22%7D,%22editorMode%22:%22builder%22%7D%5D,%22range%22:%7B%22from%22:%221726368346123%22,%22to%22:%221726386426966%22%7D%7D%7D&schemaVersion=1&orgId=1

### Steps to Reproduce

```Markdown
RUN DAILY REGRESSION ON TKE
```

**Comment by XuPeng-SH** [comment]
seems it was introduced by #18800

**Comment by XuPeng-SH** [comment]
it was introduced by #18784

**Comment by LeftHandCold** [comment]
fixed

**Comment by LeftHandCold** [comment]
fixed as #18825

**Comment by aressu1985** [comment]
fixed
